frame/balances/src/benchmarking.rs

```diff
@@ -218,5 +248,5 @@ benchmarks_instance_pallet! {
 		Balances,
 		crate::tests_composite::ExtBuilder::default().build(),
```

Comment:
Looks like it is not possible to write it without the `;` anymore, could you re-add that?  
I dont remember why, but it was desired in the MR adding it.

Reply:
looks like it works if I change the `(`, `)` to curly braces. Maybe that's what you were thinking of? It needs to be a `Stmt` so..